### PR TITLE
Extra color schemes

### PR DESCRIPTION
**Pull request by varemenos**
Could you please review the changes (apart from the changes related to the color schemes), I'm not sure whether they are valid or not

### CODE REVIEW

**Comment by Stibbons** [comment]
please rebase. Look quite good

File: src/guake/prefs.py [hunk 1]
```diff
@@ -388,12 +995,6 @@ def on_use_default_font_toggled(self, chk):
         """
         self.client.set_bool(KEY('/general/use_default_font'), chk.get_active())
 
-    def on_use_palette_font_and_background_color_toggled(self, chk):
```
Comment (Stibbons) [inline src/guake/prefs.py]: please let this method

File: src/guake/prefs.py [hunk 2]
```diff
@@ -551,14 +1152,6 @@ def toggle_style_sensitivity(self, chk):
         """
         self.get_widget('font_style').set_sensitive(not chk.get_active())
 
-    def toggle_use_font_background_sensitivity(self, chk):
```
Comment (Stibbons) [inline src/guake/prefs.py]: and this one

File: src/guake/prefs.py [hunk 3]
```diff
@@ -601,7 +1194,7 @@ def on_palette_color_set(self, btn):
         """Changes the value of palette in gconf
         """
         palette = []
-        for i in range(18):
```
Comment (Stibbons) [inline src/guake/prefs.py]: 18 is the right value

File: src/guake/prefs.py [hunk 4]
```diff
@@ -795,12 +1388,6 @@ def load_configs(self):
         self.get_widget('use_default_font').set_active(value)
         self.get_widget('font_style').set_sensitive(not value)
 
-        # use font and background color
-        value = self.client.get_bool(KEY('/general/use_palette_font_and_background_color'))
```
Comment (Stibbons) [inline src/guake/prefs.py]: please let this feature

**Comment by varemenos** [comment]
Thanks, I was at work so I couldn't reply. I will take a look at them tomorrow and let you know.

**Comment by varemenos** [comment]
Ok, so I tried rebasing the branch, could you please review it again?

File: src/guake/prefs.py [hunk 5]
```diff
@@ -848,14 +1420,6 @@ def load_configs(self):
         self.set_palette_name(value)
         self.set_palette_colors(value)
 
-        # cursor shape
```
Comment (Stibbons) [inline src/guake/prefs.py]: please review your rebase, you are removing my recent changes

use git pull --rebase

**Comment by Stibbons** [comment]
I cherry pick your patch and make the necessary correction and merge

**Comment by Stibbons** [comment]
I have merge your patch, fixes the bad rebase, and pushed. So I close your Pull Request. So thanks a lot for your effort. Can you check if I have not forgot something from your PL?

However, is it normal to have so many theme with bright background color? (I mean, because i feel more confortable with dark background).

Next step is for me, provide a convinient previsualisation for the user

**Comment by varemenos** [comment]
Ok, thanks for taking the time to fix my broken PR. Sorry for not doing it myself but I was too busy at work. I will try find some free time and research a good way to present the color schemes to the user

**Comment by Stibbons** [comment]
I Will Work on it. Can you check the color coherence?

Le jeudi 23 avril 2015, Adonis K. <notifications@github.com> a écrit :

> Ok, thanks for taking the time to fix my broken PR. Sorry for not doing it
> myself but I was too busy at work. I will try find some free time and
> research a good way to present the color schemes to the user
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Guake/guake/pull/504#issuecomment-95519174>.
>


-- 
-----
Gaetan

**Comment by varemenos** [comment]
Yes, I will do it tonight

**Comment by Stibbons** [comment]
Please look how I reorganised palette definition, this should be quite easier to add or modify palettes :)

**Comment by varemenos** [comment]
Will do. I will also update the palette string generator I'm using.

**Comment by Stibbons** [comment]
Yes, it is a dictionary now, with names and colors in the same file ! :)

**Comment by Stibbons** [comment]
Thanks again for your work

**Comment by varemenos** [comment]
I just checked the colors of the `palette.py` dictionary, everything is fine.

**Pull request closed without merging**